web/app/js/components/TapQueryForm.jsx

```diff
@@ -17,6 +17,15 @@ import { defaultMaxRps, httpMethods, tapQueryProps, tapQueryPropType } from './u
 const colSpan = 5;
 const rowGutter = 16;
 
+// you can also tap resources to tap all pods in the resource
+const resourceTypes = [
+  "deployment",
```

Comment:
No, because `shortNameLookup` includes `authority`, `service` and `namespace` which should not be included in this dropdown. I could go through and remove them, but seems like a lot of special casing.

Reply:
Ahh, good point.